### PR TITLE
Show req message

### PR DESCRIPTION
**Pull request by babybunn**

### CODE REVIEW

**Review by evilpeach — CHANGES_REQUESTED** [review]
image
Let's use circle loading instead.

**Review by evilpeach — CHANGES_REQUESTED** [review]

File: src/components/oracle-script/OracleScriptExecuteResponse.re [hunk 1]
```diff
@@ -52,34 +51,34 @@ let make = (~txResponse: TxCreator2.tx_response_t, ~schema: string) =>
...
+                     }
+                   />
+                 </div>
                </div>
-             </div>
-             <OracleScriptExecuteProof id />
-           </>;
+               <OracleScriptExecuteProof id />
+             </>
+           | None => <> <RequestFailedResult id /> </>
```
Comment (evilpeach) [inline src/components/oracle-script/OracleScriptExecuteResponse.re:80]: There is a case that ResolveStatus == Success and the result can't be decoded by OBI
https://github.com/bandprotocol/cosmoscan/blob/main/src/pages/RequestIndexPage.re#L570

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Comment by babybunn** [comment]
The screenshot shows the output as "schema not found" just in the case of Resolve status equal to "Success" but the Obi can't be decoded.

File: src/components/oracle-script/OracleScriptExecuteResponse.re [hunk 2]
```diff
@@ -91,12 +122,13 @@ let make = (~txResponse: TxCreator2.tx_response_t, ~schema: string) =>
...
-                 reportedValidators={request.reportsCount}
-                 minimumValidators={request.minCount}
-                 requestValidators={request.askCount}
+                 reportedValidators=reportsCount
+                 minimumValidators=minCount
+                 requestValidators=askCount
                />
              </div>
            </div>
+         | Some({resolveStatus: _, id}) => <RequestFailedResult id />
```
Comment (evilpeach) [inline src/components/oracle-script/OracleScriptExecuteResponse.re:131]: It's no need to use `resolveStatus: _`

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by evilpeach — APPROVED** [review]

**Pull request merged**